server/src/main/java/org/opensearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -906,6 +924,52 @@ static Settings aggregateIndexSettings(
         return indexSettings;
     }
 
+    /**
+     * Updates index settings to enable remote store by default based on cluster level settings
+     * @param settingsBuilder index settings builder to be updated with relevant settings
+     * @param requestSettings settings passed in during index create request
+     * @param clusterSettings cluster level settings
+     */
+    private static void updateRemoteStoreSettings(Settings.Builder settingsBuilder, Settings requestSettings, Settings clusterSettings) {
+        if (CLUSTER_REMOTE_STORE_ENABLED_SETTING.get(clusterSettings)) {
+            // Verify if we can create a remote store based index based on user provided settings
+            if (!canCreateRemoteStoreIndex(requestSettings)) {
```

Comment:
nit: `canCreateRemoteStoreIndex(requestSettings) == false` (Applicable to all the places where we are using `!` in if condition)

Reply:
Sure